change/@fluentui-react-native-experimental-radio-group-346f9bef-d345-48b5-b108-de0a1146e8fd.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is this relevant to this PR? It looks like an old changelong? This comment doesn't seem to deal with this PRs new feature

Reply:
Thanks for catching this. I think it carried over from an older local branch that I had forked from, which is also why there are so many old commits in the history. Will fix this for future PRs by forking directly from the main branch.